tests/async_client_test.py

```diff
@@ -370,6 +370,21 @@ def test_register(self, async_client, aioresponse):
         assert isinstance(resp, RegisterResponse)
         assert async_client.access_token
 
+    def test_deactivate(self, async_client, aioresponse):
+        loop = asyncio.get_event_loop()
+
+        assert not async_client.access_token
+
+        aioresponse.post(
+            "https://example.org/_matrix/client/r0/admin/deactivate",
+            status=200,
+            payload=self.DeactivateResponse
+        )
+        resp = loop.run_until_complete(async_client.deactivate("user", "admin_token"))
```

Comment:
No need to use `loop.run_untli_complete()` if you convert this to an async function.

Reply:
I believe I have resolved